src/platforms/javascript/common/sourcemaps/uploading/angular-webpack.mdx

```diff
@@ -0,0 +1,115 @@
+---
+title: Angular CLI
+description: "Upload your source maps using the Angular CLI, a custom Angular builder and the Sentry Webpack plugin."
+sidebar_order: 0
+supported:
+  - javascript.angular
+---
+
+You can use the Angular CLI together with our Sentry Webpack plugin to automatically create releases and upload source maps to Sentry when building your app (e.g. with `ng build`).
+Due to Angular's closed build system, you first need to to configure a custom builder to register the Webpack plugin.
+
+## Installation
+
+```shell {tabTitle:npm}
+npm install --save-dev @angular-builders/custom-webpack @sentry/webpack-plugin
+```
+
+```shell {tabTitle:yarn}
+yarn add --dev @angular-builders/custom-webpack @sentry/webpack-plugin
+```
+
+## Configuration
+
+First, make sure that you're [generating source maps](../../.#generating-source-maps) in your Angular project.
+Then, set up a [custom Angular builder](https://www.npmjs.com/package/@angular-builders/custom-webpack), to which you add the Sentry Webpack plugin.
+
+### 1. Set up custom Angular builder
+
+In your `angular.json`, replace the default builder (`@angular-devkit/build-angular`) with `@angular-builders/custom-webpack`:
+
+```javascript {filename:angular.json}
+{
+  "projects": {
+    "my-project": {
+      "architect": {
+        "build": {
+          "builder": "@angular-builders/custom-webpack:browser",
+          "options": {
+            "customWebpackConfig": {
+              "path": "./webpack.config.js"
+            },
+            // ... other options
+          },
+          // ... other options
+        },
+        // ... other options
+      },
+      // ... other options
+    }
+  }
+}
+```
+
+Note: This will have no effect on your development server by using `ng serve`. Only `ng build` is configured to use the custom builder and the Webpack plugin. If you're using an Angular version below 13
+
```

Comment:
```suggestion
### 1. Set Up Custom Angular Builder

In your `angular.json`, replace the default builder (`@angular-devkit/build-angular`) with `@angular-builders/custom-webpack`:

```javascript {filename:angular.json}
{
  "projects": {
    "my-project": {
      "architect": {
        "build": {
          "builder": "@angular-builders/custom-webpack:browser",
          "options": {
            "customWebpackConfig": {
              "path": "./webpack.config.js"
            },
            // ... other options
          },
          // ... other options
        },
        // ... other options
      },
      // ... other options
    }
  }
}
```

<Alert title="Note" level="info">

If you're using any Angular version below 13, using `ng serve` will have no effect on your development server. Only `ng build` is configured to use the custom builder and the Webpack plugin. 

<Alert>

```

Not sure if the whole note applies to Angular versions below 13 or just the second part...?

Reply:
Ooops, I totaly forgot to finish the last sentence, which should just mention that the build command is slightly different for NG13 and below.

Reply:
Done [f5d6de2](https://github.com/getsentry/sentry-docs/pull/6404/commits/f5d6de24350c97288998ee7a69ce8c5fbc596181)